File: apps/bacteriaquantify/lib/models/Bacteries.dart
class Bacteries {
  int? iD;
  String? species;
  int? totalColony;

  Bacteries({this.iD, this.species, this.totalColony});

  Bacteries.fromJson(Map<String, dynamic> json) {
    iD = json['ID'];
    species = json['species'];
    totalColony = json['totalColony'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['ID'] = this.iD;
    data['species'] = this.species;
    data['totalColony'] = this.totalColony;
    return data;
  }
}

File: apps/rest-server/src/dto/Bacteries.go
package dto

type Bacteries struct {
	ID          int    `json:"ID, omitempty"`
	Species     string `gorm:"column:species;not null;default:''::character varying;comment:NGS generated" json:"species"` // NGS generated
	TotalColony int32  `gorm:"column:totalColony;not null;comment:NGS generated" json:"totalColony"`                       // NGS generated
}
